pkg/manifest/account.go

```diff
@@ -0,0 +1,97 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package manifest
+
+import (
+	"errors"
+	"fmt"
+	"github.com/google/uuid"
+)
+
+var (
+	errNameMissing   = errors.New("name is missing")
```

Comment:
why introduce unexported sentinel errors? Are they just used in tests?

Reply:
Yes. I wanted to not rely on the exact messages of errors but wanted to test them properly.